src/rpc/quorums.cpp

```diff
@@ -592,7 +592,7 @@ static RPCHelpMan quorum_verify()
     }
 
     uint256 quorumHash(ParseHashV(request.params[4], "quorumHash"));
-    llmq::CQuorumCPtr quorum = llmq_ctx.qman->GetQuorum(llmqType, quorumHash);
+    auto quorum = llmq_ctx.qman->GetQuorum(llmqType, quorumHash);
```

Comment:
if you changed this line, can add `const` also, because why not

Reply:
Resolved in latest push!